tensorboard/plugins/projector/vz_projector/vz-projector-inspector-panel.ts

```diff
@@ -282,6 +365,10 @@ namespace vz_projector {
       this.searchBox.message = '';
       const minDist = neighbors.length > 0 ? neighbors[0].dist : 0;
 
+      if (this.spriteImagesAvailable && this.showNeighborImages) {
+        var image_renderer = this.spriteImageRenderer();
```

Comment:
nit: Rename to `imageRenderer` (camel case).

Reply:
too much time in python recently lol